apps/site/pages/en/learn/getting-started/fetch.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Node.js Fetch
+layout: learn
+authors: benhalverson, LankyMoose
+---
+
+# Using the Fetch API with Undici in Node.js
+
+## Introduction
+
+[Unidici](httpss://undici.nodejs.org) is an HTTP client libary that powers the fetch API in Node.js. It was written from scratch and does not rely on the built-in HTTP client in Node.js. It includes a number of features that make it a good choice for high-performance applications.
+
+```mjs
+async function main() {
+  const response = await fetch('https://jsonplaceholder.typicode.com/posts');
+  const data = await response.json();
+  console.log(data);
+}
+
+main().catch(console.error);
+```
+
+## Customizing the Fetch API with Undici
+
+Undici allows you to customize the Fetch API by providing options to the `fetch` function. For example, you can set custom headers, set the request method, and set the request body. Here is an example of how you can customize the Fetch API with Undici:
+
+With a pool, you can reuse connections to the same server, which can improve performance. Here is an example of how you can use a pool with Undici:
+
+```mjs
+import { Pool } from 'undici';
+
+const ollamaPool = new Pool('http://localhost:11434', {
```

Comment:
Shouldn't we aim to use `fetch` instead?

Reply:
Yes I am. This is an advanced example of using Undici. Undici is the library that Node.js uses to make fetch work.

Reply:
I added a get and post example using fetch above this.